packages/auth/src/security.js

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
Moving this file to central auth package so that it can be used by both meditrak-server and report-server